src/MiNET/MiNET/Net/RakNet/RakConnection.cs

```diff
@@ -673,7 +676,7 @@ internal async Task UpdateAsync(RakSession session)
 
 					long elapsedTime = datagram.Timer.ElapsedMilliseconds;
 					long datagramTimeout = rto * (datagram.TransmissionCount + session.ResendCount + 1);
-					datagramTimeout = Math.Min(datagramTimeout, 3000);
+					//datagramTimeout = Math.Min(datagramTimeout, 3000);
```

Comment:
Did this cause issues?

Reply:
Packet loss from the client can cause to unexpected disconnection.

Reply:
Hmm, i still feel like it'd be good to atleast have A timeout to avoid filling memory with crap. Maybe make it configurable using the config.

Reply:
Yes, you're right, I will.